src/web.ts

```diff
@@ -1,17 +1,196 @@
-import { WebPlugin } from '@capacitor/core';
+import { WebPlugin } from "@capacitor/core";
 
-import { CapacitorFirebaseDynamicLinksPlugin, LinkConfig } from './definitions';
+import { CapacitorFirebaseDynamicLinksPlugin, LinkConfig } from "./definitions";
 
-export class CapacitorFirebaseDynamicLinksWeb extends WebPlugin implements CapacitorFirebaseDynamicLinksPlugin {
-  createDynamicLink(_: LinkConfig): Promise<{ value: string; }> {
-    throw new Error('Method not implemented.');
+interface DynamicLinkInfo {
+  longDynamicLink: string;
+  suffix?: {
```

Comment:
Am I blind or do you not use this anymore?
Please remove this, if this is the case.

Reply:
The thing is, that the REST API allows to provide a suffix parameter. That's why I added it to the interface. However, the suffix settings is not used in any of the current code. Hence, it is not set.

What do you recon? Remove it or keep it for reference?

Reply:
Ah okay, that makes sense. I actually don't look into the docs before.